content/community/contributing/handbook/front-end/developing/_index.md

```diff
@@ -48,17 +48,17 @@ weight: 11
 
 {{% notice info %}}
 Note that you can also run Altinn Studio Designer frontend in the same way as running app frontend. This would require changing the 
-`src/studio/src/designer/backend/views/ServiceDevelopment/Index.cshtml` (for app-development) or `src/studio/src/designer/backend/views/Home/Index.cshtml` 
+`backend/src/Designer/views/ServiceDevelopment/Index.cshtml` (for app-development) or `backend/src/Designer/views/Home/Index.cshtml` 
```

Comment:
Maybe add a link to the part about localhost:8080 (on a different page) here?

Reply:
Does this still work as before or should we suggest using our new method with the .env file?

Reply:
Good point. I think it would still work as long as you stop the altinn-designer container as described. But to make it easier to maintain the docs I have updated this with a reference to our readme on how to run the solutions locally.